docs/designers-developers/designers/block-design.md

```diff
@@ -17,7 +17,7 @@ Basic block settings won’t always make sense in the context of the placeholder
 
 ### The block sidebar should only be used for advanced, tertiary controls
 
-The sidebar is not visible by default on a small / mobile screen, and may also be collapsed in a desktop view. Therefore, it should not be relied on for anything that is necessary for the basic operation of the block. Pick good defaults, make important actions available in the block toolbar, and think of the sidebar as something that only power users may discover. In addition, use sections and headers in the block sidebar if there are more than a handful of options, in order to allow users to easily scan and understand the options available.
+The sidebar is not visible by default on a small / mobile screen, and may also be collapsed in a desktop view. Therefore, it should not be relied on for anything that is necessary for the basic operation of the block. Pick good defaults, make important actions available in the block toolbar, and think of the sidebar as something that most users should not need to open. In addition, use sections and headers in the block sidebar if there are more than a handful of options, in order to allow users to easily scan and understand the options available. Each block sidebar comes with an "Advanced" panel by default. This panel houses an "Additional CSS Class" field, and should be used to house other power user controls. 
```

Comment:
```suggestion
The sidebar is not visible by default on a small / mobile screen, and may also be collapsed in a desktop view. Therefore, it should not be relied on for anything that is necessary for the basic operation of the block. Pick good defaults, make important actions available in the block toolbar, and think of the sidebar as something that most users should not need to open.

In addition, use sections and headers in the block sidebar if there are more than a handful of options, in order to allow users to easily scan and understand the options available. Each block sidebar comes with an "Advanced" section by default. This area houses an "Additional CSS Class" field, and should be used to house other power user controls. 
```

I think it'd be good to break up this paragraph; it's getting a bit long. I also tweaked it to say "section" instead of "panel" because we don't really use the term panel elsewhere in this section of the document; it'd be good to have introduced and defined that term before using it.

Reply:
Also… is it possible to put arbitrary controls into the Advanced panel? I don't think I've seen any examples in the wild; if it _is_ possible, we should definitely make sure it's documented.

Reply:
I'd go a step further and make "Each block sidebar comes with..." its own paragraph as well.

@youknowriad do you know the answer to @chrisvanpatten's question about adding arbitrary controls into the Advanced panel?

Reply:
@youknowriad @melchoyce Dug into this a bit and it is indeed possible to add controls here via the `InspectorAdvancedControls` slotfill component :)